generated/crd/tackle.konveyor.io_addons.yaml

```diff
@@ -49,20 +49,6 @@ spec:
                 - Always
                 - Never
                 type: string
-              mounts:
-                description: Mounts optional.
-                items:
-                  description: Mount specification.
-                  properties:
-                    claim:
-                      type: string
-                    name:
-                      type: string
-                  required:
-                  - claim
-                  - name
-                  type: object
-                type: array
```

Comment:
Support for arbitrary volumes to be mounted in addon pods has no supporting use case. This existed to support maven m2 cache.
This has been replaced by the shared _cache_ concept.